src/v2/Components/Message/AssistantMessageEntry.tsx

```diff
@@ -22,23 +22,27 @@ export const AssistantMessageEntry: FunctionComponent<AssistantMessageProps> = (
   if (message.options) {
     return <AssistantButtonEntry message={message} ask={ask} preview={preview} blockInput={blockInput} />;
   }
+
   return (
     <PFAssistantMessageEntry icon={ChatbotIcon}>
       <TextEntry content={message.content} preview={preview} />
     </PFAssistantMessageEntry>
   );
 };
+
 export const AssistantButtonEntry: FunctionComponent<AssistantMessageProps> = ({ message, ask, preview, blockInput }) => {
   return (
     <PFAssistantMessageEntry
       options={message.options?.map((o, index) => ({
-        title: o.value ?? '',
+        title: o.text ?? '',
```

Comment:
Instead of the value in the button, lets use the text that we got from the server as the label/title of the button.